packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -170,11 +168,11 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
     const {
       actions,
       dismissible,
-      dismissAction,
+      dismissAction = {},
       content,
```

Comment:
I would rather suggest to fix `propTypes` and use for these `shorthandAllowingChildren`

Reply:
Great, thanks... Didn't know about `shorthandAllowingChildren`

Reply:
Done